### DISCUSSION TITLE
Can PnP modern search v3.0 and PnP modern search v4.0 webparts co-exist in same tenant ?

### DISCUSSION THREAD

**Discussion by hsamdani12** [microsoft-search/pnp-modern-search#2319] [Q&A] [answered]
We have v3.0 installed and being used at couple of places. Can I install the 4.0 as well in same tenant to use for new project ? Will it have any issues ?

**Comment by wobba** [comment, marked as answer]
Yes :) I would try not to have them on the same page, but that should work as well.

**Comment by Heavm00** [comment]
Both versions are working great ;-) I'm using both versions in case of a migration szenario.

**Reply by M365specialist** [reply]
Thank you so much, I was just about to install V4 alongside V3 and it's good to know that you have already done it without issues.
